src/System.Diagnostics.PerformanceCounter/ref/System.Diagnostics.PerformanceCounter.cs

```diff
@@ -30,7 +30,7 @@ public partial struct CounterSample
         public static bool operator ==(System.Diagnostics.CounterSample a, System.Diagnostics.CounterSample b) { throw null; }
         public static bool operator !=(System.Diagnostics.CounterSample a, System.Diagnostics.CounterSample b) { throw null; }
     }
-    public sealed partial class PerformanceCounter
+    public sealed partial class PerformanceCounter : System.ComponentModel.Component, System.ComponentModel.ISupportInitialize
```

Comment:
So we build it but don't package it... where do we use it?  Does it ship anywhere?  If not, why do we have it in the repo at all?